ait/dsn/sle/cltu.py

```diff
@@ -324,6 +324,23 @@ def decode(self, message):
         '''
         return super(self.__class__, self).decode(message, CltuProviderToUserPdu())
 
+    def _replace_v4_pdus(self):
+        '''Replaces a given version 5 asn1 definition class with version 4 class'''
+
+        def replace(clsname):
+            modname = 'ait.dsn.sle.pdu.cltu.cltuv4'
+            module = pydoc.locate(modname)
+            if module is None:
+                raise ImportError('No module named %s' % modname)
+            replcls = getattr(module, clsname)
+            if replcls is None:
+                raise ImportError('No class named %s.%s' % (modname, clsname))
+            ait.core.log.info("Replacing %s with class %s" % (clsname, replcls))
+            globals()[clsname] = replcls
+
+        replace('CltuUserToProviderPdu')
+        replace('CltuUserToProviderPdu')
```

Comment:
Is this duplicate call intentional or was this meant to be a UserToProvider and ProviderToUser swap?